src/components/button/button.scss

```diff
@@ -31,6 +31,15 @@
     min-width: initial;
   }
 
+  &.btn-float {
+    box-shadow: var(--kd-elevation-level-3);
+  }
+  &.btn-hidden {
+    opacity: 0;
+    visibility: hidden;
+    transition: opacity 0.5s ease, visibility 0s 0.5s;
```

Comment:
.5s is a bit slow. We're still working on the foundation animation module, but maybe go with `300ms ease-in-out` for now.